mcs_benchmark_data/models/benchmark_dev_dataset.py

```diff
@@ -1,10 +1,22 @@
 from dataclasses import dataclass
 from dataclasses_json import LetterCase, dataclass_json
 
+from rdflib import Graph
+from rdflib.resource import Resource
+from ..namespace import 
+
 from mcs_benchmark_data.models.benchmark_dataset import BenchmarkDataset
 
 @dataclass_json(letter_case=LetterCase.CAMEL)
 @dataclass(frozen = True)
 class BenchmarkDevDataset(BenchmarkDataset):
     '''A dataset containing dev samples of a benchmark'''
-    type: "BenchmarkDevDataset"
\ No newline at end of file
+    type: "BenchmarkDevDataset"
+
+    def to_rdf(
+        self, *, graph: Graph, **kwds) -> Resource:
+        super.to_rdf(self,graph=graph)
+
+        #How to add type?

```

Comment:
You're already doing RDF.type in the superclass, with this class's name as its type.